addon/bootstrap.js

```diff
@@ -99,13 +93,17 @@ async function startup(addonData, reason) {
     });
   }
 
-  console.log(new Feature().startup());
+  // just fire this, then done.
+  console.log(new Feature(variation).start());
+  // also set a time to handle "long expiration"
+  // TODO glind
 
 }
 
 
 function shutdown(addonData, reason) {
   console.log("shutdown", studyUtils.REASONS[reason] || reason);
+  const { REASONS } = studyUtils;
```

Comment:
What is this doing here? Are you shortening the path from `studyUtils.REASONS` to just `REASONS`? Kind of like saying: `const REASONS = studyUtils.REASONS`?

Reply:
Yes.  this is called "object destructuring" in es6.